content/css/sel4.css

```
@@ -266,4 +266,15 @@ a.divLink {
   color:#B78B41;  /* emphasisDark */
   font-size:15px;
   font-weight:bold;
-}
\ No newline at end of file
+}
+
+.emph {
+    color:#FFCA5F;
```

I'd try this one here:

```suggestion
    color:#B78B41;  /* emphasisDark */
```